.github/workflows/ci.yaml

```diff
@@ -86,41 +90,52 @@ jobs:
           #   fails multiple times while others don't, its cache will likely get evicted,
           #   making it even slower to test and fix (uffff).
           #   When they fix this, we should remove ${{ github.run_id }} from the end of the key.
-          key: wasp-build-${{ runner.os }}-${{ matrix.ghc }}-${{ hashFiles('waspc/waspc.cabal') }}-${{ hashFiles('waspc/cabal.project') }}-${{ github.run_id }}
+          key: wasp-build-${{ runner.os }}-${{ matrix.ghc }}-${{ hashFiles('waspc/waspc.cabal') }}-${{ hashFiles('waspls/waspls.cabal') }}-${{ hashFiles('cabal.project') }}-${{ github.run_id }}
           restore-keys: wasp-build-${{ runner.os }}-${{ matrix.ghc }}-
 
-      - name: Check Haskell code formatting
+      - name: "Check Haskell code formatting"
         if: matrix.os == 'ubuntu-latest'
-        run: ./run ormolu:check
+        uses: mrkkrp/ormolu-action@v5  # ormolu 4.0.0.
```

Comment:
I had this idea that we will use local `run` script to run these dev tools, but it now take me some time to adapt it to work for waspls also now that we have two packages, so I just went for easier option which is ormolu gh action, that uses same version as we have specified in the project and that is it. This is good enough for now.